app/src/main/java/net/bible/android/view/util/buttongrid/ButtonGrid.kt

```diff
@@ -218,14 +212,13 @@ class ButtonGrid constructor(context: Context, attrs: AttributeSet? = null, defS
 
     private fun showPreview(buttonInfo: ButtonInfo) {
         try {
-            val button = buttonInfo.button
             if (buttonInfo != currentPreview) {
-                Log.d(TAG, "Previewing " + buttonInfo.name)
+                Log.d(TAG, "Previewing " + buttonInfo.description)
                 currentPreview = buttonInfo
-                previewText!!.text = buttonInfo.name
+                previewText!!.text = buttonInfo.description
                 val popupHeight = previewHeight
                 previewText!!.measure(MeasureSpec.makeMeasureSpec(0, MeasureSpec.UNSPECIFIED), MeasureSpec.makeMeasureSpec(0, MeasureSpec.UNSPECIFIED))
-                val popupWidth = Math.max(previewText!!.measuredWidth, buttonInfo.button!!.width + previewText!!.paddingLeft + previewText!!.paddingRight)
+                val popupWidth = max(previewText!!.measuredWidth, buttonInfo.button!!.width + previewText!!.paddingLeft + previewText!!.paddingRight)
```

Comment:
Android Studio shows the `!!` after `.button` is useless, you can remove that as well